### ISSUE TITLE
odd date parsing behaviour

### ISSUE THREAD

**Issue by jclulow** [davepacheco/node-jsprim#8]
If I pass in the string `"1"` to `parseDateTime()`, I get what seems to be a reasonable result:

```
> jsprim.parseDateTime('1').toISOString()
'1970-01-01T00:00:00.001Z'
```

But if I pass in the string `"0"`...

```
> jsprim.parseDateTime('0').toISOString()
'2000-01-01T08:00:00.000Z'
```

**Comment by davepacheco** [comment]
This looks like a silly falsiness bug.  Sorry about that.

**Comment by davepacheco** [comment]
@jclulow, could you review:
https://github.com/davepacheco/node-jsprim/compare/dev-issue-8

**Comment by jclulow** [comment]
LGTM!

**Comment by davepacheco** [comment]
Published as v1.2.1.
